requirements/edx/base.in

```diff
@@ -146,6 +147,7 @@ sortedcontainers                    # Provides SortedKeyList, used for lists of
 sqlparse                            # Required by Django to run migrations.RunSQL
 stevedore                           # Support for runtime plugins, used for XBlocks and edx-platform Django app plugins
 staff-graded-xblock                 # An XBlock that allows for off-site bulk scoring of problems
+super-csv                           # Generic CSV processor
```

Comment:
I do just want to point out that we don't need to explicitly list this as a dependency; it'll be pulled in as a dep of edx-bulk-grades if we don't